buildchain/buildchain/targets/remote_image.py

```diff
@@ -90,17 +90,34 @@ def _show(task: types.Task) -> str:
         return utils.title_with_target1('IMG PULL', task)
 
     def _build_actions(self) -> List[types.Action]:
+        """Compute actions for the image - pull, tag, save."""
         filepath = self.uncompressed_filename
+        pull_command = PullCommand(
+            repository=self.repository, digest=self.digest
+        )
+        if self.for_containerd:
```

Comment:
```Python
# Repository is tag without the version.
repository = self.tag[:self.tag.rfind(':')]`
```

Reply:
Not accurate afaict, we need more informations than just tag with `self.repository`. Maybe I'm looking at the wrong line?

Reply:
renaming to local_repo for a more accurate description.